db/schema.rb

```diff
@@ -1,57 +1,57 @@
-# This file is auto-generated from the current state of the database. Instead
```

Comment:
Uh....what happened here? This file shouldn't getting rewritten—make sure you're not changing \n to \r\n because windows
See: https://help.github.com/articles/dealing-with-line-endings/
And the 'dos2unix' command